src/start/RealmScreen.js

```diff
@@ -24,7 +24,7 @@ type Props = $ReadOnly<{|
   // `navigation` prop for free, with the stack-nav shape.
   navigation: NavigationStackProp<{|
     ...NavigationStateRoute,
-    params?: {|
+    params: {|
```

Comment:
We only navigate to the 'realm' route with `navigateToRealmScreen`
    and `resetToRealmScreen`, and in both cases, we pass `params`.

There's now another case, effectively: `getInitialRouteInfo`. The conclusion is still true, though.

(Also `resetToRealmScreen` now has no callers, so we could delete it.)

Reply:
Similarly the next commit's message:

    In `resetToRealmScreen` and `navigateToRealmScreen`, the only two
    things we call to navigate to the 'realm' route, neither `realm` nor